ci-operator/step-registry/ipi/conf/ibmcloud/custom-endpoints/ipi-conf-ibmcloud-custom-endpoints-commands.sh

```diff
@@ -44,6 +44,9 @@ EOF
   yq-go m -a -x -i "${CONFIG}" "${config_patch}"
 }
 
+if [ -n "$SERVICE_ENDPOINT_COS" ]; then
```

Comment:
If not moving this block, what would happen ?

Reply:
https://issues.redhat.com/browse/OCPBUGS-46601
ref https://github.com/openshift/installer/blob/d2bb75079b1d73ed533c3b998550090893237bd5/pkg/types/ibmcloud/metadata.go#L51-L56 
changed the order

Reply:
I do not understand why it has to force users to change the endpoints config order in install-config.yaml, shouldn't installer set the arguments correctly whatever endpoints order configured by users ?

Per my understanding of https://github.com/openshift/installer/blob/d2bb75079b1d73ed533c3b998550090893237bd5/pkg/types/ibmcloud/metadata.go#L51-L56, it just will change the parameter formats automatically for the 1st endpoint, but not care about what is the 1st endpoint. <region>:<endpoint-1>=<endpoint-1-url>,<endpoint-2>=<endpoint-2-url>,...,<endpoint-N>=<endpoint-N-url>